ui/src/main/java/com/epam/eco/kafkamanager/ui/topics/browser/handlers/FilterOperationMapHandler.java

```diff
@@ -0,0 +1,158 @@
+/*******************************************************************************
+ *  Copyright 2023 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License.  You may obtain a copy
+ *  of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ *******************************************************************************/
+package com.epam.eco.kafkamanager.ui.topics.browser.handlers;
+
+import java.util.Map;
+
+import org.apache.avro.util.Utf8;
+
+import com.epam.eco.kafkamanager.FilterClause;
+
+import static com.epam.eco.kafkamanager.ui.topics.browser.FilterClauseUtils.KEY_VALUE_SEPARATOR;
+
+/**
+ * @author Mikhail_Vershkov
+ */
+
+public class FilterOperationMapHandler extends FilterOperationHandler<Map<Object,Object>> {
```

Comment:
I am afraid of huge memory impact while creating new instance of handlers like this for any filtering.

Reply:
It is very hard to do. Because every handler depends from concrete clause. So you can't initialize it in the beginning.